config/routes.rb

```diff
@@ -415,7 +415,7 @@
   get "/users/:username/curated_clusters" => "curated_clusters#index"
   get "/users/:username/curated_clusters/new" => "curated_clusters#new"
   get "/users/:username/curated_clusters/:id" => "curated_clusters#show"
-  post "/users/:username/curated_clusters" => "curated_clusters#create"
+  post "/users/:username/curated_clusters" => "curated_clusters#addarticle"
```

Comment:
gets changed later

Reply:
Yep this got deleted. Great catch, updatd so we don't conflict with previous code